var/spack/repos/builtin/packages/libglvnd/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libglvnd(AutotoolsPackage):
+    """The GL Vendor-Neutral Dispatch library
+
+    libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API
+    calls between multiple vendors. It allows multiple drivers from different
+    vendors to coexist on the same filesystem, and determines which vendor to
+    dispatch each API call to at runtime.
+
+    Both GLX and EGL are supported, in any combination with OpenGL and OpenGL
+    ES."""
+
+    homepage = "https://github.com/NVIDIA/libglvnd"
+    url      = ("https://github.com/NVIDIA/libglvnd/releases/download"
+                "/v1.1.1/libglvnd-1.1.1.tar.gz")
+    git      = "https://github.com/NVIDIA/libglvnd.git"
+
+    version('master', branch='master')
+
+    version('1.1.1', sha256='71918ed1261e4eece18c0b74b50dc62c'
+                            '0237b8d526e83277ef078554544720b9')
```

Comment:
This can be on a single line, Spack ignores checksums during flake8 line length checks.